src/lang/terms/holds.pl

```diff
@@ -206,3 +207,64 @@
 data_base_type_(_Property,DataValue,string) :- string(DataValue),!.
 data_base_type_(_Property,true,bool) :- !.
 data_base_type_(_Property,false,bool) :- !.
+
+     /*******************************
+     *	    UNIT TESTS	     		*
+     *******************************/
+
+:- use_module(library('lang/query'),
+  [ tell/1]).
+
+:- use_module(library('lang/terms/temporal')).
+:- use_module(library('db/tripledb_tests')).
+:- use_module(library('db/tripledb'),
+  [ tripledb_load/2 ]).
+
+:- use_module('holds.pl').
+
+:- begin_tripledb_tests(
+    'lang_holds',
+    'package://knowrob/owl/test/swrl.owl',
+    [ namespace('http://knowrob.org/kb/swrl_test#')
+    ]).
+
+
+test('ask triples holds') :-
+  assert_true(holds(test:'Ernest', test:'hasSibling', test:'Fred')).
+
+test('ask holds with arity 1') :-
+  assert_true(holds(test:'hasHeightInMeters'(test:'RectangleBig',13.5))).
```

Comment:
this works now? else you can disable the test as in:

    test('<(Event6,Short4)', [ fixme('support mixing of quantitative+qualitative input') ]) :-
	    interval_before(test:'Event6', test:'Short4').